browser/brave_wallet/ethereum_provider_impl_unittest.cc

```diff
@@ -1680,9 +1681,7 @@ TEST_F(EthereumProviderImplUnitTest, SignMessage) {
               &error_message);
   EXPECT_TRUE(signature.empty());
   EXPECT_EQ(error, mojom::ProviderError::kUnauthorized);
-  EXPECT_EQ(error_message,
-            l10n_util::GetStringFUTF8(IDS_WALLET_ETH_SIGN_NOT_AUTHED,
-                                      base::ASCIIToUTF16(address)));
+  EXPECT_EQ(error_message, l10n_util::GetStringUTF8(IDS_WALLET_NOT_AUTHED));
```

Comment:
Using a generic `IDS_WALLET_NOT_AUTHED` message when `CheckAccountAllowed` fails, applies to all test changes in this file(and other test files)